### ISSUE TITLE
inverted FlatList focus issue

### ISSUE THREAD

**Issue by ArturasMi** [react-native-community/react-native-tvos#30]
When you focus FlatList item that has inverted set to true it flips item 180 deg. It's a known issue for react-native https://github.com/facebook/react-native/issues/21919 but it looks like "This issue seems to have a really low activity on it" it so I decided to write it here since react-native-tvos is focusing on Apple TV.

React Native version:
  IDEs:
    Xcode: 11.1/11A1027 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0
    react-native-tvos:  0.61.2-0
  npmGlobalPackages:
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7

## Steps To Reproduce

Just create a FlatList with prop: inverted={true}

Describe what you expected to happen:
To focus without a flip

**Comment by dlowder-salesforce** [comment]
@ArturasMi thanks for the reminder about this issue.  I just took a look.  It appears that you can work around this immediately by commenting out line 217 of RCTTVView.m as shown below.  Looks like the FlatList inversion is doing its own affine transform that conflicts with the one in RCTTVView.  I'll start working on a fix.

```objc
 [UIView animateWithDuration:0.2 animations:^{
          NSLog(@"transform = %@", NSStringFromCGAffineTransform(self.transform));
          // self.transform = CGAffineTransformMakeScale(1, 1);
      }];
```

**Comment by dlowder-salesforce** [comment]
@ArturasMi could you test with the changes in the above commit and verify that the issue is resolved?

**Comment by ArturasMi** [comment]
@dlowder-salesforce Hi, thanks for your quick reply. Yes I tested and I can confirm that your suggestion above worked!

**Comment by dlowder-salesforce** [comment]
Fixed in release 0.61.5-0.
